src/pages/office/office.ts

```diff
@@ -264,6 +258,7 @@ export class OfficePage extends BasePageComponent {
       this.pageState.weatherConditions$,
       this.pageState.d255$,
       this.pageState.additionalInformation$,
+      this.pageState.identification$,
```

Comment:
Why do we have to subscribe to all these observables? There is not particular action which needs to be executed.